packages/design-system/src/components/Dropdown/Select.test.tsx

```diff
@@ -13,120 +14,131 @@ const defaultProps: SelectProps = {
+
+    const select = screen.getByRole('combobox');
+    expect(select).toHaveClass('ds-c-field--small');
   });
 
-  it('adds size classes to select element', () => {
-    const mediumData = render({ size: 'medium' });
-    const smallData = render({ size: 'small' });
+  it('adds medium size classes to select element', () => {
+    makeSelect({ size: 'medium' });
 
-    expect(mediumData.wrapper.hasClass('ds-c-field--medium')).toBe(true);
-    expect(smallData.wrapper.hasClass('ds-c-field--small')).toBe(true);
+    const select = screen.getByRole('combobox');
+    expect(select).toHaveClass('ds-c-field--medium');
   });
 
   it('is inversed', () => {
-    const data = render({ inversed: true });
+    makeSelect({ inversed: true });
 
-    expect(data.wrapper.hasClass('ds-c-field--inverse')).toBe(true);
+    const select = screen.getByRole('combobox');
+    expect(select).toHaveClass('ds-c-field--inverse');
   });
 
   it('has error', () => {
-    const data = render({ errorMessage: 'Error' });
+    makeSelect({ errorMessage: 'Error' });
 
-    expect(data.wrapper.prop('aria-invalid')).toBe(true);
-    expect(data.wrapper.hasClass('ds-c-field--error')).toBe(true);
+    const select = screen.getByRole('combobox');
+    expect(select).toHaveAttribute('aria-invalid', 'true');
+    expect(select).toHaveClass('ds-c-field--error');
   });
 
   it('handles bottom placed error', () => {
-    const data = render({
+    makeSelect({
       errorMessage: 'Error',
       errorPlacement: 'bottom',
       errorId: '1_error',
     });
 
-    expect(data.wrapper.prop('aria-invalid')).toBe(true);
-    expect(data.wrapper.prop('aria-describedby')).toBe('1_error');
-    expect(data.wrapper.hasClass('ds-c-field--error')).toBe(true);
-    expect(data.wrapper).toMatchSnapshot();
+    const select = screen.getByRole('combobox');
+    expect(select).toHaveAttribute('aria-invalid', 'true');
+    expect(select).toHaveAttribute('aria-describedby', '1_error');
+    expect(select).toHaveClass('ds-c-field--error');
+    expect(select).toMatchSnapshot();
   });
 
   it('is disabled', () => {
-    const data = render({ disabled: true });
+    makeSelect({ disabled: true });
 
-    expect(data.wrapper.prop('disabled')).toBe(true);
+    const select = screen.getByRole('combobox');
+    expect(select).toHaveAttribute('disabled');
   });
 
   it('is not disabled', () => {
-    const data = render();
+    makeSelect();
 
-    expect(data.wrapper.prop('disabled')).toBeUndefined();
+    const select = screen.getByRole('combobox');
+    expect(select).not.toHaveAttribute('disabled');
   });
 
-  describe('event handlers', () => {
-    const wrapper = render().wrapper;
-
-    beforeEach(() => {
-      jest.clearAllMocks();
-    });
-
-    it('calls the onChange handler', () => {
-      wrapper.simulate('change');
-      expect(defaultProps.onBlur).not.toHaveBeenCalled();
-      expect(defaultProps.onChange).toHaveBeenCalled();
-    });
+  it('calls the onChange handler', () => {
```

Comment:
I pulled your branch down locally and tested - those blur/change event tests passed. I even passed wrong expectations and they failed, so it seems to be working now?